### ISSUE TITLE
NWB: displaying session metadata

### ISSUE THREAD

**Issue by bendichter** [flatironinstitute/neurosift#4]
![image](https://github.com/flatironinstitute/neurosift/assets/844306/ce68f937-1fe1-4e4f-8ff1-9edbe8edf4e6)

metadata such as session_start_time and identifier are displayed on the left. Would it be possible to remove these inactive entries from the according on the right?

**Comment by magland** [comment]
@bendichter 

To address this, I have created a new button called "Browse NWB File" toward the bottom. That opens a new internal tab that allows browsing of the groups and datasets of the nwb file. This is useful for exploring in the case where there is content that neurosift doesn't yet support.

With this new feature, I felt comfortable removing the datasets from the main tab. I also reorganized the top level headings in the main tab. They don't necessarily correspond to the top level groups. The main tab will evolve to reflect the types of views supported.

![image](https://github.com/flatironinstitute/neurosift/assets/3679296/b53934d2-156a-4d67-8d1f-00cfcce8f369)

![image](https://github.com/flatironinstitute/neurosift/assets/3679296/45dbf117-d349-4232-b89a-4a60ad469f0d)
